### DISCUSSION TITLE
Oberheim Matrix 1k Ctrlr VST Ableton 10 Live

### DISCUSSION THREAD

**Discussion by prple1** [RomanKubiak/ctrlr#381] [Q&A]
Hi there,

I got back to the oberheim matrix 1000 ctrlr in ableton live. I have updated it, as well. But the still have the same problem. 

This is my old post:

"Hi there,

Soooo I got myself an Oberheim Matrix 1000 and upgraded it to version 1.20 as I wanted to control it from Ableton live 10.

I want to use the Ctrlr VST (https://ctrlr.org/tag/matrix-1000/) The VST it is already installed and running.

I use Windows 10 and a Behringer FCA1616 interface. The driver I use in Ableton to connect the interface is the ASIO one.

Now, I have two questions/ problems to solve.

1. The VST window is not fully visible in Ableton Live, I can only see 2/3 of it and there is now option to scale it up or down or to move around in any way. Anyone else had the same problem? - solved

2. How do I setup the VST routing to control the matrix parameters while sending midi notes to the matrix? 
Edit: There is an Interface between the Matrix and Ableton. I am currently using two synths that are controlled by Ableton. The Midi Output of my Interface goes into the Midi in of the Matrix and the out by "Thru" to another Synth's Midi in.  I cannot simply use the Interface as Output device. It alway gives me an error message. 

Hope someone can help! 

Best

**Comment by dnaldoog** [comment]
Hi there @prple1 

it sounds like you are using some  version 6 of Ctrlr. 

Version 5.3.201 should fix the windowing issue you are having, so you will need to recompile the vst of course.

I don’t know about Ableton, but you may be able to connect using instructions ( for Cubase ) found here https://ctrlr.org/roland-jd990-super-editor/

Regards,

**Reply by dobo365** [reply]
The point is that the panel is provided by possemo as fully prepared VST and that he compiled it with the v6.x version as stated on his web site.
As we don't have access to the source panel, we cannot change the code (add a zoom function) or recompile it in 5.3.201 (I'm only using 5.3.201 myself - no issue).
So, @prple1, you need to contact possemo either thru the old Ctrlr site or thru his website. Friendly guy so I'm sure he will help you.

**Comment by Godlike-Productions** [comment]
On windows you need to disable the MIDI port in Ableton and connect it in the VST plugin directly.  Then make sure Thru Host -> Out is enabled to pass MIDI notes from ableton thru the plugin to the MIDI port.

For zoom Ctrl - and Ctrl + should always work for zooming in and out.
